src/components/menu/__internal__/submenu/submenu.component.js

```diff
@@ -43,7 +43,7 @@ const Submenu = React.forwardRef(
     const menuContextOpen = menuContext.openSubmenu;
 
     const formattedChildren = React.Children.map(children, (child) => {
-      if (child.type === ScrollableBlock) {
+      if (child && child.type === ScrollableBlock) {
```

Comment:
I guess this change not corresponding to the PR title and I guess it should be as a different commit at least in this case

Reply:
It can't be actually because removing the check for `SubmenuBlock` in `MenuItem` causes this to error, so the commit would be broken if I split it out

Reply:
More specifically removing the `React.Children.map` bit on line 45 means that null children can now (correctly) be passed into `Submenu` and this extra check is needed. It should have been there anyway